### DISCUSSION TITLE
Small error in the Bevy Book

### DISCUSSION THREAD

**Discussion by HotSammiches** [bevyengine/bevy#2512] [Q&A] [answered]
In going through the Bevy Book I was following the simple example and ran into a build error on this page: https://bevyengine.org/learn/book/getting-started/ecs. I'm not sure who to contact about this so I thought I'd start here. 

Here is the code in question:

```
fn greet_people(query: Query<&Name, With>) {
    for name in query.iter() {
        println!("hello {}!", name.0);
    }
}
```
I fixed it by changing the `With` to `With<Person>`. I was actually glad it failed because the code made more sense after fixing it than it did before! The text of the book made it clear what the query was doing so it was an easy fix.

**Comment by alice-i-cecile** [comment]
In general, issues with the website belong in the [`bevy-website` repo](https://github.com/bevyengine/bevy-website) :)

Looking at the source though, this error seems to be corrected already: https://github.com/bevyengine/bevy-website/blob/master/content/learn/book/getting-started/ecs/_index.md.

From @MinerSebas, "The HTML sees the <Person> as an HTML Tag"

![image](https://user-images.githubusercontent.com/3579909/126523567-fbcf171e-7d96-445b-a109-a76cb1c6ee0c.png)

So, good eye! This may be more complex to fix 🤔

**Reply by alice-i-cecile** [reply]
And of course, `<Person>` disappeared from my comment above too...

**Reply by alice-i-cecile** [reply]
Filed an issue with Zola: https://github.com/getzola/zola/issues/1555

**Comment by mockersf** [comment, marked as answer]
It's an issue in the website configuration due to the release of newest version of Zola that deprecates an option that we were using.

Thanks for noticing and reporting! It should be fixed by https://github.com/bevyengine/bevy-website/pull/191

**Comment by HotSammiches** [comment]
Thanks for the reply! I'm newish to Rust and very new to Bevy so I appreciate you guys getting back to me so quickly.

**Reply by mockersf** [reply]
It's fixed now on the website 🙂
